py/qqa/webpages/templates/logfile.html

```diff
@@ -0,0 +1,59 @@
+{% extends "qabase.html" %}
+{% block body %}
+
+<style>
+label {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  text-align: right;
+  width: 400px;
+  line-height: 26px;
+  margin-bottom: 10px;
+}
+
+input {
+  height: 20px;
+  flex: 0 0 200px;
+  margin-left: 10px;
+}
+
+table {
+  border-collapse: collapse;
+}
+
+.container{
+  display: flex;
+}
+.flex-item{
+  flex-grow: 1;
+}
+</style>
```

Comment:
Are HTML files allowed to have multiple <style> definition blocks?  Are they allowed in the <body>?  `base.html` also has a style block between the <head> and the <body>.  Maybe this is perfectly fine, or maybe it violates the standard and browsers are just being forgiving.  Please check.

Reply:
It extends from both, but I think some restructuring will be good anyway. The preproc image nav tables also use the same style options that I copy-pasted, so perhaps a parent template can serve all of them